docs/data/charts/scatter/scatter.md

```diff
@@ -60,9 +60,31 @@ As with other charts, you can modify the [series color](/x/react-charts/styling/
 You can also modify the color by using axes `colorMap` which maps values to colors.
 The scatter charts use by priority:
 
-1. The y-axis color
-2. The x-axis color
-3. The series color
+1. The z-axis color
+2. The y-axis color
+3. The x-axis color
+4. The series color
+
+:::info
+The z-axis is a third axis that allows to customize scatter points independently from their position.
```

Comment:
Is this an API you saw on another library?
It's clearly a valid use-case, but I'm curious to know if this is the classic way of solving it

Reply:
The naming is used in Rechart

https://recharts.org/en-US/examples/BubbleChart

```jsx
<ZAxis type="number" dataKey="value" domain={domain} range={range} />
```


For Echart, they have a similar idea called [visualMap](https://echarts.apache.org/en/option.html#visualMap)